elements/lisk-p2p/src/p2p.ts

```diff
@@ -154,7 +154,6 @@ export class P2P extends EventEmitter {
 	private readonly _handleInboundSocketError: (error: Error) => void;
 	private readonly _peerHandshakeCheck: P2PCheckPeerCompatibility;
 
-	// tslint:disable-next-line: cyclomatic-complexity
 	public constructor(config: P2PConfig) {
```

Comment:
Can we organize all the initialization based on category? like it is for `constants.ts`, we can have `// Event handlers`, `Config`, `PeerPool`, `PeerBook`, etc